### ISSUE TITLE
Change startpage to rooms

### ISSUE THREAD

**Issue by NPadrutt** [Odonno/modern-gitter-winjs#30]
Currently the start page is the "welcome to gitter" page. While I think it's a good idea to have such a page where you can introduce your self I wouldn't define this as the start page.
First doesn't it offer any information I need when I use the app, second after the first time I saw it I don't get any new information. Therefore after the first 3-5 starts it's just an annoying click to get to my rooms.

I would move this Menu point to the bottom of the list with menu entries, set rooms to the top and make this the default page since joining a existing room probably will be the command a user will use the most.

**Comment by Odonno** [comment]
It references #6 issue.

Moreover, what you need is maybe links from the home page ? I don't know what is the best option but clearly, implementing #6 will save you a lot of time.

**Comment by NPadrutt** [comment]
I guess for me the best solution would be to open the last chat as long as the app is in the memory and if it's opened completly new to show the room overview.
But I would still change the order of the menu's. Normally I try to sort them by the order importance / usage. IMHO that makes it easier to find what you are looking for.

**Comment by Odonno** [comment]
@NPadrutt The .42 will handle the startpage feature (both rooms and room pages).

Give me your feedback.

**Comment by NPadrutt** [comment]
Imho much better. The only thing what I would change is move the home sceen away from the top spot.
for me the best spot is bottom of the screen as for example in the music app the settings or profile page

**Comment by Odonno** [comment]
I started by copying the 'Groove Music' app splitview. I don't remember clearly how it look, yes. I'll check that.

But, the home page is really an home page. I don't think it should be moved, but maybe redesigned, don't you think ?

**Comment by NPadrutt** [comment]
what is the usage scenario you have in mind? When would you go to the homepage?

**Comment by Odonno** [comment]
Well, you may be right. The home page does not provide a lot of value.
Like you did, I should remove the home icon from splitview command bar.

I already thought about a settings page, it'll come in time.
About profile page, I don't know if it's interesting. Maybe to handle multiple accounts. Do you think it's valuable ?

**Comment by NPadrutt** [comment]
I'm not sure you should remove it. but just move it away from the topspot. At the end of the list would be the "fast fix". or make two lists one for the top and one for the bottom?

I always liked an about page. You could place the basic infos, make a link to the repo (if someone wants to contribute), version, contact information, review button.. So I guess there is enough content for a dedicated "about this program" page besides an Profile page.

**Comment by Odonno** [comment]
But it will not be the same page, will it ?
I think the `home` page should be first experience of the app. I want to do something like Spotify app.
And the `about` page should contain that basic infos.

**Comment by NPadrutt** [comment]
not the same as the current one or what do you mean?

afaik spotify presents infos like playlists etc. there? what do you wanna show there in gitter?

**Comment by Odonno** [comment]
The home page has a specfic design to help people use the app the first time. Like Spotify, it shows essential features (with bullets points and clear images). Or even like Word/Excel Mobile. That's what you asked some weeks ago, no ?

So, the home page will evolve to be not JUST an about page. That's why I want to split this up now.

**Comment by NPadrutt** [comment]
and what is the value for someone who already knows the app? what you describe sounds more like an intro dialog to me.

**Comment by Odonno** [comment]
Yes, replacing home page by introduction page. The fact is we have that one life time page (dialog) at one hand, and an about page (always accessible) in the other hand.

**Comment by NPadrutt** [comment]
I would do an dialog the first time the app is opened, or until you check a tick or so and then an about page with the basic information an maybe an option to open the introdcution again or so. but no entry in the side menu.

**Comment by NPadrutt** [comment]
On the other hand.. do you really need an introduction? I mean, assume everyone who downlaods the app and creates a user / Logges in knows about what the app is and more or less how to use it.

**Comment by Odonno** [comment]
I don't know. I'll start to make an about page and we'll see what could be done.

**Comment by inf3rno** [comment]
When I try to use the link: https://gitter.im/ I always end up on https://gitter.im/travis-ci/travis-ci . Is there a way to change this?
